### PR TITLE
doc(pii): Advanced data scrubbing stabilized

### PR DESCRIPTION
**Pull request by untitaker**
This is the absolute minimum amount of changes we need to get out on June 30th.

### CODE REVIEW

File: src/collections/_documentation/data-management/advanced-datascrubbing.md [hunk 1]
```diff
@@ -1,25 +1,15 @@
 ---
-title: 'Advanced Data Scrubbing (Beta)'
+title: 'Advanced Data Scrubbing'
 sidebar_order: 4
 keywords: ["pii", "gdpr", "personally identifiable data", "compliance"]
 ---
 
-In addition to using [`beforeSend`]({% link _documentation/data-management/sensitive-data.md %}#custom-event-processing-in-the-sdk) in your SDK or our [server-side data scrubbing features]({% link _documentation/data-management/sensitive-data.md %}#server-side-scrubbing) to redact sensitive data, we are currently beta-testing ways to give you more granular control over server-side data scrubbing of your events. Additional functionality includes:
+In addition to using [`beforeSend`]({% link _documentation/data-management/sensitive-data.md %}#custom-event-processing-in-the-sdk) in your SDK or our [regular server-side data scrubbing features]({% link _documentation/data-management/sensitive-data.md %}#server-side-scrubbing) to redact sensitive data, Advanced Data Scrubbing is an alternative way to redact sensitive information just before it is saved in Sentry. It allows you to:
```
Comment (PeloWriter) [inline src/collections/_documentation/data-management/advanced-datascrubbing.md:7]: the {% link...} approach is apparently problematic - might wish to use the markdown format here.

File: src/collections/_documentation/data-management/advanced-datascrubbing.md [hunk 2]
```diff
@@ -40,7 +30,7 @@ Rules generally consist of three parts:
 ## Methods
 
 - _Remove_: Remove the entire field. We may choose to either set it to `null`, remove it entirely or replace it with an empty string depending on technical constraints.
```
Comment (PeloWriter) [inline src/collections/_documentation/data-management/advanced-datascrubbing.md:32]: nit: serial comma (add a comma after "remove it entirely")

File: src/collections/_documentation/data-management/advanced-datascrubbing.md [hunk 3]
```diff
@@ -40,7 +30,7 @@ Rules generally consist of three parts:
...
-- _Mask_: Replace all characters with `*`. For creditcards this replaces everything but the last 4 digits.
+- _Mask_: Replace all characters with `*`.
 - _Hash_: Replace the matched substring with a hashed value.
 - _Replace_: Replace the matched substring with a constant placeholder value such as `[Filtered]` or `[creditcard]`. Right now this value cannot be configured.
```
Comment (PeloWriter) [inline src/collections/_documentation/data-management/advanced-datascrubbing.md:35]: Suggest replacing "Right now" with "Currently" (it's what we're using in other places, so easier to find if we're searching and making changes. But it's not critical.

Comment (untitaker) [reply src/collections/_documentation/data-management/advanced-datascrubbing.md]: the entire sentence was incorrect, removed

**Comment by untitaker** [comment]
thanks @PeloWriter for reviewing early!

File: src/collections/_documentation/data-management/advanced-datascrubbing.md (hunk 1 above)
Comment (untitaker) [reply src/collections/_documentation/data-management/advanced-datascrubbing.md]: yeah, replaced

**Comment by untitaker** [comment]
GA toggle PR is here: https://github.com/getsentry/sentry/pull/19630

**Pull request merged**